core/templates/pages/story-editor-page/modal-templates/story-editor-unpublish-modal.component.html

```diff
@@ -1,70 +1,78 @@
 <div *ngIf="isSerialChapterFeatureFlagEnabled()">
   <div class="modal-header">
+    <span *ngIf="unpublishedChapters.length === 1">Unpublishing Chapter</span>
+    <span *ngIf="unpublishedChapters.length > 1">Unpublishing Chapters</span>
+  </div>
+  <div class="modal-body">
     <p>
-      You're attempting to unpublish chapters, please specify a reason and note the warning
+      You're attempting to unpublish
+      <span *ngIf="unpublishedChapters.length === 1">chapter</span>
+      <span *ngIf="unpublishedChapters.length > 1">chapters</span>
+      <span *ngFor="let chapter of unpublishedChapters"> {{chapter}},</span>
```

Comment:
Please read [this comment](https://github.com/oppia/oppia/pull/18740#discussion_r1284414638) carefully, this doesn't match what it says.

Note that in "7-8", "7" is the first chapter and "8" is the last chapter. If chapters 7, 8 and 9 were unpublished then it should be "7-9".

Reply:
Done